.changeset/netlify-skew-protection.md

```diff
@@ -0,0 +1,8 @@
+---
+'astro': minor
+'@astrojs/netlify': minor
+---
```

Comment:
I think this should be separate changesets - one for the core support, one for skew protection in Netlify, and a patch for fixing skew protection in Vercel